### PR TITLE
Created form

### PR DESCRIPTION
**Pull request by idungstanley**
Collaborated with Andres in working on my portfolio project as required, we worked on a single branch and this is the result we got.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -216,6 +216,90 @@ <h4>Skills</h4>
...
+                name="message"
+                id="message"
+                cols="30"
+                rows="11"
+                maxlength="500"
+                placeholder="Enter text here"
+                required
+              ></textarea>
+              </label>
+            </li>
```
Comment (Deepakdanger) [inline index.html:265]: - [ ] The width of the `textarea` is not 100%, I recommentd to use width 100% in style .
-
Screenshot from 2022-03-22 23-40-36

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @idungstanley, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (idungstanley) [reply index.html]: the style width was 100%, I changed the id name which was what affected my whole item, making the styles not be effective on it, thanks for the review! it is all checked now.

Comment (Whoistolu) [reply index.html]: All good now 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @idungstanley 

Great job implementing the recommended changes 👍🏽 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**